getambassadorio_blc.py

```diff
@@ -174,16 +174,8 @@ def product_ugly_check(self, link: Link) -> None:
     @staticmethod
     def _parse_srcset_value(attrvalue: str) -> List:
         delimiter = ','
-        if "&rect=" in attrvalue and "https://cdn.sanity.io/images/" in attrvalue:
-            links = []
-            tokens = attrvalue.split(',', 4)
-            while len(tokens) >= 4:
-                links.append(delimiter.join(tokens[0:4]).split(' ')[0])
-                attrvalue = attrvalue[
-                    attrvalue.find('h', len(delimiter.join(tokens[0:4]))) :
-                ]
-                tokens = attrvalue.split(delimiter, 5)
-            return links
+        if "?rect=" in attrvalue and "https://cdn.sanity.io/images/" in attrvalue:
```

Comment:
I think it would be better to keep both `?rect=` and `&rect=` like this
```python
if ("?rect=" in attrvalue or "&rect=" in attrvalue) and "https://cdn.sanity.io/images/" in attrvalue:
```